src/TourManager/Repository/Repositories/TourRepository.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Travely.TourManager.Repository.Repositories.Entities;
+
+namespace Travely.TourManager.Repository.Repositories
+{
+    class TourRepository : GenericRepository<Tour>
+    {
+        private TourContext _context;
```

Comment:
You can make GenericRepository more generic and make also DbContext Type as argument to avoid keeping to copy of context in one repository